docs/releases/5.0.txt

```diff
@@ -37,6 +37,14 @@ compatible with Django 5.0.
 What's new in Django 5.0
 ========================
 
+Facet filters in the admin
+--------------------------
+
+Facet counts are now shown for applied filters in the admin changelist when
+toggled on via the UI. This behavior can be changed via the new
+:attr:`.ModelAdmin.show_facets` attribute. For more information see
```

Comment:
Maybe the toggle in the UI should be referred to as headline feature rather than the `ModelAdmin` attribute?

Reply:
@smithdc1 Any wording suggestion?

Reply:
I must have completely miss-read this earlier today 🤦.

The first sentence references the UI so I think it's fine as it is (for some reason I thought this section made no reference to the UI toggle at all). 

Sorry.